lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
FYI, this is not what array-like means in any other scientific projects in the community. The fact that scatter, bar, step use array-like to mean scalar or array-like is probably just a mistake.